addons/mail_bot/static/src/scss/discuss.scss

```diff
@@ -0,0 +1,21 @@
+.o_NotificationAlert {
+     display: flex;
+     flex-flow: column;
+     align-items: center;
+}
+
+.o_NotificationAlert_url {
+     margin-top: 16px;
+     padding: 8px;
+     width: 75%;
+     position: relative;
+     font-style: italic;
+     text-align: left;
+}
+
+.o_NotificationAlert_urlClipboardButton {
+     padding: 8px;
+     position: absolute;
+     top: 0;
+     right: 0;
+}
```

Comment:
No absolute positioning. From the look of it, this should be replaced with horizontal flex layout.